src/components/WebLogView/Row.tsx

```diff
@@ -8,24 +8,44 @@ import { TableCellWithTooltip } from '../TableCellWithTooltip'
 import { HighlightedText } from '../HighlightedText'
 import { getRequestType } from './WebLogView.utils'
 import { SearchResults } from './SearchResults'
+import { useMemo } from 'react'
 
 interface RowProps {
   data: ProxyDataWithMatches
   isSelected?: boolean
   onSelectRequest: (data: ProxyDataWithMatches) => void
   filter?: string
+  highlightedRequestIds?: string[]
```

Comment:
Note: I think we're approaching a moment when we'll need to make WebLogView more modular and to essentially have 2 separate components for Generator (where we need highlighted requests) and Recorder (when we add some recording editing functionality perhaps?)

Reply:
Agree, I've added ability to inject Row component through props and removed `highlightedRequestIds` dependency from WebLogView